src/nih_wayfinding/app/scripts/profiles/profile-service.js

```diff
@@ -0,0 +1,56 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function ProfileService (ProfileProvider) {
+
+        // Private variables
+        var profilesList = [];
+
+        // Public Interface
+        var module = {
+            getProfile: getProfile,
+            getProfileNames: getProfileNames,
+            createProfile: createProfile
+        };
+
+        return module;
+
+        /**
+         * @returns Profile for given username
```

Comment:
Convention seems to be to put docstrings immediately before or within a function definition.  Presumably if we were ever to run an automated documentation generator against it, that's where it would expect it to be.